prototype_source/flight_recorder_tutorial.rst

```diff
@@ -0,0 +1,130 @@
+(prototype) Flight Recorder for Debugging
```

Comment:
Can you please add the following sections:
- What you will learn
- Prerequisites
- Conclusion
Please follow this template: 
https://github.com/pytorch/tutorials/blob/main/beginner_source/template_tutorial.py